### DISCUSSION TITLE
When will support for plugins begin?

### DISCUSSION THREAD

**Discussion by gnusec** [fox0430/moe#2121] [General]
Plugins are very important. For example, I want to perform text sorting, deduplication, or generate queues, etc. Would you consider using nimscript or a compiled version to implement this? Or would you use SDL?"

**Comment by fox0430** [comment]
I was thinking of using Nimscript or Lua to implement the plugin system, but I'm currently focused on LSP support and I'm not sure if I'll actually work on this.
